docker-entrypoint-funcs.sh

```diff
@@ -1,10 +1,16 @@
 #!/usr/bin/env bash
 
+declare -A log_levels=([trace]=0, [info]=1, [warning]=2)
+_script_log_level="info"
```

Comment:
```suggestion
declare -A _edbdocker_log_levels=([trace]=0, [info]=1, [warning]=2)
_edbdocker_log_level="info"
```

This file is intended to be `source`-d into other scripts, so all variables and functions must be namespaced carefully.